python/ray/train/_internal/state/state_actor.py

```diff
@@ -47,5 +44,16 @@ def get_or_create_state_actor():
         ).remote()
 
     # Ensure the state actor is ready
-    ray.get(state_actor.ready.remote())
+    ray.get(state_actor.__ray_ready__.remote())
     return state_actor
+
+
+def get_state_actor():
```

Comment:
```
def get_state_actor() -> Optional[ActorHandle[TrainStateActor]]:
```

Reply:
Nice catch. But seems that `ActorHandle` is not subscriptable since it's not a generic class. I've changed to `def get_state_actor() -> Optional[ActorHandle]:`